content/efcore/getting-started/tutorial-model-reference.md

```diff
@@ -1,7 +1,6 @@
 # Entity Classes 
- 
-Here is what the entity classes that make up our example data model look like:
- 
+This document contains class definitions of the example data models used throughout this tutorial.
```

Comment:
It's one single data model. The entity classes make up the data model. 
Maybe, "This document contains the entity class definitions for the example data model used throughout this tutorial."